syntax = "proto3";

package mavlink.msg.common;

message CommandLong {
  float param1 = 1; /*<  Parameter 1 (for the specific command)*/
  float param2 = 2; /*<  Parameter 2 (for the specific command)*/
  float param3 = 3; /*<  Parameter 3 (for the specific command)*/
  float param4 = 4; /*<  Parameter 4 (for the specific command)*/
  float param5 = 5; /*<  Parameter 5 (for the specific command)*/
  float param6 = 6; /*<  Parameter 6 (for the specific command)*/
  float param7 = 7; /*<  Parameter 7 (for the specific command)*/
  uint32 command = 8; /*<  Command ID (of command to send)*/
  uint32 target_system = 9; /*<  System which should execute the command*/
  uint32 target_component = 10; /*<  Component which should execute the command, 0 for all components*/
  uint32 confirmation = 11; /*<  0: First transmission of this command 1-255: Confirmation transmissions (e.g. for kill command)*/
}

